### DISCUSSION TITLE
Any way to Build with Metal/OpenCL support on Mac?

### DISCUSSION THREAD

**Discussion by nkleinsasser2** [dlemstra/Magick.NET#1765] [Help]
I built the ImageMagick library dylib's with OpenCL support, however, I cannot seem to build Magick.NET OpenMP for macOS. Is this even possible or am I continuing to waste my time?

Any advice is greatly appreciated.

Ultimate Goal: GPU accelerated operations on MacOS for Magick.NET.

**Comment by dlemstra** [comment]
I have not done the research yet so I cannot give you an answer. If you have added support for OpenCL would you mind creating a PR for that in the Magick.Native repository? And if you manage to get OpenMP to work please mention this issue: https://github.com/dlemstra/Magick.Native/issues/27.
